src/test/authClass.login.spec.ts

```diff
@@ -0,0 +1,47 @@
+import { expect } from 'chai'
+import * as sinon from 'sinon'
+
+import { AuthService } from '../apiServices/AuthService'
+import { JwtTokensInterface } from '..'
+
+describe('loginUser', () => {
+    const tokens: JwtTokensInterface = {
+        access: 'access',
+        refresh: 'refresh',
+    }
+
+    let loginUserStub: any
+    let AUTH
+    beforeEach(() => {
+        AUTH = new AuthService('user', 'password')
+        loginUserStub = sinon.stub(AUTH, 'login')
```

Comment:
It looks like we're not actually testing login here. Because we are stubbing login, we're not actually testing it's functionality. We want to stub the functions that login is using, then call the real `AUTH.login()`, and check that login behaves correctly.

(same with logout)